config/deployment/kustomization.yaml

```diff
@@ -0,0 +1,11 @@
+namespace: control-api
+
+bases:
+- ../namespace
+- ../rbac
```

Comment:
AFAIK `bases` is deprecated. The lines should just go to `resources` down below.

Reply:
You're right. I somehow remember this not working in the `kubectl` provided version. But it seems to work fine now.